addons/web/static/src/views/kanban/kanban_compiler.js

```diff
@@ -1,90 +1,102 @@
+import { archParseBoolean } from "@web/views/utils";
+import { ViewCompiler } from "@web/views/view_compiler";
 
-/**
- * @typedef {Object} DropdownDef
- * @property {Element} el
- * @property {boolean} inserted
- * @property {boolean} shouldInsert
- * @property {("dropdown" | "toggler" | "menu")[]} parts
- */
-
-const ACTION_TYPES = ["action", "object"];
-const SPECIAL_TYPES = [...ACTION_TYPES, "edit", "open", "delete", "url", "set_cover", "archive", "unarchive"];
+const SPECIAL_TYPES = ["edit", "delete", "archive", "unarchive", "set_cover"];
 
 export class KanbanCompiler extends ViewCompiler {
     setup() {
-        this.ctx.readonly = "read_only_mode";
         this.compilers.push(
-            { selector: ".oe_kanban_colorpicker", fn: this.compileColorPicker },
-            { selector: "t[t-call]", fn: this.compileTCall },
-            { selector: "img", fn: this.compileImage }
+            { selector: "kanban", fn: this.compileKanban, doNotCopyAttributes: true },
+            { selector: "kanban > header", fn: () => false }, // do nothing
+            { selector: "kanban > progressbar", fn: () => false }, // do nothing
+            { selector: "kanban > control", fn: () => false } // do nothing
         );
     }
 
     //-----------------------------------------------------------------------------
     // Compilers
     //-----------------------------------------------------------------------------
 
-    /**
-     * @override
-     */
-    compileButton(el, params) {
-        /**
-         * WOWL FIXME
-         * For some reason, buttons in some arch have a data-something instead of just a normal attribute.
-         * The new system only uses normal attributes.
-         * This is an ugly small compatibility trick to fix this.
-         */
-        if (el.hasAttribute("data-type")) {
-            for (const { name, value } of el.attributes) {
-                el.setAttribute(name.replace(/^data-/, ""), value);
+    compileKanban(el, params) {
+        let withAside = false;
+        const kanban = createElement("div");
+        const sectionContainer = createElement("div");
+        sectionContainer.setAttribute(
+            "class",
+            "o_kanban_section_container d-flex flex-column justify-content-between gap-2 w-100 h-100"
+        );
+        for (const child of el.childNodes) {
+            if (getTag(child) === "aside") {
+                withAside = true;
+                append(kanban, this.compileAside(child, params));
+            } else if (getTag(child) === "section") {
+                append(sectionContainer, this.compileSection(child, params));
+            } else if (getTag(child) === "menu") {
+                append(kanban, this.compileMenu(child, params));
+            } else {
+                append(kanban, this.compileNode(child, params));
             }
         }
+        append(kanban, sectionContainer);
+        kanban.setAttribute("class", `w-100${withAside ? " d-flex flew-row" : ""}`);
```

Comment:
a little inoffensive typo here
```suggestion
        kanban.setAttribute("class", `w-100${withAside ? " d-flex flex-row" : ""}`);
```